en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+ 
+ - **API Usage By Backend**
+ 
+    This widget shows the per-backend breakdown of requests for the APIs you selected. This is useful when multiple APIs are sharing the same backend, which has traffic restrictions. You can use these stats to proactively scale your backends. You can switch between the two views using the small icon at the upper-right corner of the widget.
+ 
+    ![API usage by backend](../assets/img/insights/usage-by-backend.png){.cInlineImage-half}
+ 
+ - **API Resource Usage**
+ 
+    This table shows a resource-level breakdown of API traffic. Each row represents an API resource and it shows the API name, resource path, API method, and the hit count for that combination.
+    
+    ![API resource usage](../assets/img/insights/resource-usage.png){.cInlineImage-half}
+    
+### Errors
+
+This page shows information related to erroneous API calls that are received by your system. The errors are categorization based on the error type. You can further drill down using the error subtypes. Use this page as the starting point for debugging any API errors.
+
+![Error statistics](../assets/img/insights/error-page-full.png){.cInlineImage-full}
+
+You can filter the information displayed in the widgets as follows:
+
+| **Filtering Option** | **Description** |
+|--------------------|-------------------------------------------------------------------------------------------------|
+| **By API** | In the **API** field, select the APIs for which you want to view analytics. By default, all APIs are selected. You can select one or multiple APIs from this selector and view the aggregated result. |
+| **By Category** | The **Category** field is displayed when you click **Category** in the toggle switch in the upper right corner of the page. In this field, you can select one or all of the following error categories:<br/>**Authentication**: Any kind of authentication error falls into this category including expired/missing/invalid credentials.<br/>**Target Connectivity**: Any kind of API back-end error falls into this category including connection time-outs/other back-end errors (for example, 4xx and 5xx status codes).<br/>**Throttling**: Any kind of API back-end error falls into this category including application throttling/subscription throttling.<br/>**Other**: All other errors fall into this category including mediation errors/resource not found errors. |
+| **By Status Code** | The **Status Code** field is displayed when you click **Status Code** in the toggle switch in the upper right corner of the page. In this field, you can select all or any available HTTP status code category (4xx, 5xx, etc.). These status codes represent the response status of the API Gateway. |
+
+#### Category mode
+
+The following widgets are available for monitoring errors when you have selected the category mode via the toggle switch in the upper-right corner of the page.
```

Comment:
shall we make "category mode" bold?
also should we say the catergory mode? we are talking about an input value right?

Reply:
It is not an input value. Used the word to refer to the mode in which data is displayed when **Category** switch button is selected.

Reply:
shall we not refer to it as "category mode"?

Reply:
Changed